colors.css

```diff
@@ -0,0 +1,19 @@
+/* Light theme */
```

Comment:
I like your idea of moving those two files to a styles folder.  I would probably call the css template something more like "whitespace-colors-css.tmpl".   Pretty common extension for identifying templates :)